Assets/HoloToolkit/UX/Scripts/Collections/ObjectCollectionDynamic.cs

```diff
@@ -0,0 +1,155 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace HoloToolkit.Unity.Collections
+{
+    /// <summary>
+    /// A utility that stores transform information for objects in a collection
+    /// This info can then be used for non-destructive realtime manipulation
+    /// Or to restore an earlier configuration gathered on startup
+    /// </summary>
+    public class ObjectCollectionDynamic : MonoBehaviour
+    {
+        public enum BehaviorEnum
+        {
+            StoreEveryTime,
+            StoreOnceOnStartup,
+            StoreManually,
+        }
+
+        /// <summary>
+        /// Extends collection node class to include stored local position / rotation data
+        /// </summary>
+        [Serializable]
+        public class CollectionNodeDynamic : CollectionNode
+        {
+            public CollectionNodeDynamic(CollectionNode node)
+            {
+                transform = node.transform;
+                Name = node.Name;
+                Offset = node.Offset;
+                Radius = node.Radius;
+
+                localPositionOnStartup = transform.localPosition;
+                localEulerAnglesOnStartup = transform.localEulerAngles;
+            }
+
+            public CollectionNodeDynamic (Transform nodeTransform)
+            {
+                transform = nodeTransform;
+                Name = nodeTransform.name;
+                Offset = Vector2.zero;
+                Radius = 0f;
+
+                localPositionOnStartup = transform.localPosition;
+                localEulerAnglesOnStartup = transform.localEulerAngles;
+            }
+
+            public Vector3 localPositionOnStartup;
+            public Vector3 localEulerAnglesOnStartup;
+        }
+
+        /// <summary>
+        /// When to gather the dynamic node information
+        /// </summary>
+        public BehaviorEnum Behavior = BehaviorEnum.StoreEveryTime;
+
+        /// <summary>
+        /// List of dynamic nodes in the collection
+        /// </summary>
+        public List<CollectionNodeDynamic> DynamicNodeList;
+
+        [SerializeField]
+        private ObjectCollection collection;
+
+        private bool updatedOnce = false;
```

Comment:
Hi @DDReaper,

In regards to:
> The scene starts with gaze, then shifts to pointer when a trigger is used BUT only on the hand that triggered, so you can only use one hand at a time?
> No way to go back to using gaze in the example (but this may be expected behaviour) or a limitation of only supporting 1 pointer?

This is how it's implemented in MRTK and nothing to do with this PR for basic interaction functionality.  You should open an issue on this against MRTK.

I'll look into the reference above and the warning you get with the anim button.

thanks,
-pat